src/ui/public/visualize/loader/__tests__/visualize_loader.js

```diff
@@ -59,9 +60,9 @@ describe('visualize loader', () => {
 
   function embedWithParams(params) {
     const container = newContainer();
-    loader.embedVisualizationWithSavedObject(container, createSavedObject(), params);
+    loader.embedVisualizationWithSavedObject(container[0], createSavedObject(), params);
     $rootScope.$digest();
-    return container.find('visualize');
+    return container.find('.visualize');
```

Comment:
Could you add a `data-test-subj` attribute to the element instead? This will uncouple our styles from our tests. Coupling them creates a brittle relationship and means that we'll break tests during simple CSS refactors, like renaming a class.

```js
    // visualize_loader.js, line 73
    const element = document.createElement('div');
    element.className = 'visualize';
    element.setAttribute('data-test-subj', 'visualizationLoader');
    container.appendChild(element);
```

```js
// Find this element in a test.
container.find('[data-test-subj="visualizationLoader"]');
```